tests/test_processing_templates.py

```diff
@@ -31,7 +31,6 @@ class TestParameterlessTemplates:
         [
             templates.DetectionBuildingsSpacept,
             templates.DetectionTreesSpacept,
-            templates.DetectionTreesHeightsSpacept,
```

Comment:
This change would force upgrade 3.0.0 version. We either need to postpone it or rethink our versioning rules around processing images